agent/php_wrapper.h

```diff
@@ -89,8 +89,44 @@
  *    1) before_callback gets called when OAPI triggers the begin function hook.
  *    2) after_callback gets called when OAPI triggers the end function hook.
  *    3) clean_callback gets called in the case of dangling segments that occur
- * because an exception causes the end function hook to NOT be called and thus
- * the clean function resets any variables.
+ *    4) unless explicitly setting any of the above callbacks, the default
+ * callback is set to after_callback. because an exception causes the end
+ * function hook to NOT be called and thus the clean function resets any
+ * variables.
```

Comment:
It seems like part of the additional explanation for point 3) got wrapped into the additional explanation of point 4)

Reply:
Good catch, thanks!
Please see: 
c817e7d